DDS: Squid Integration (#18715)
DataDog/integrations-core: squid/CHANGELOG.md

<!-- towncrier release notes start -->

## 3.0.0 / 2023-09-12 / Agent 7.57.0

***Changed***:

* Updated pipeline and standard remappers.

***Added***:

> * Added support for different log formats of access logs e.g common and combined.

Reviewer: ```suggestion
* Added support for different log formats of access logs, such as common and combined.
```

> * Added OOTB assets dashboard, monitors, detection rule.

Reviewer: ```suggestion
* Added an OOTB assets dashboard, monitors, and a detection rule.
```
